src/cart/crt_init.c

```diff
@@ -14,22 +14,26 @@
 #include <sys/resource.h>
 #include "crt_internal.h"
 
-struct crt_gdata crt_gdata;
-static volatile int   gdata_init_flag;
+struct crt_gdata        crt_gdata;
+static volatile int     gdata_init_flag;
```

Comment:
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt